src/components/app-configure/access/DeleteDelegate.vue

```diff
@@ -0,0 +1,79 @@
+<script lang="ts" setup>
```

Comment:
Remove this component. Is it used anywhere?

Reply:
ok

Reply:
https://github.com/arcana-network/developer-dashboard/pull/156/commits/31946dcfa27636ce0d82d033445c2b6c9a907d3b